airbyte-cdk/python/pyproject.toml

```diff
@@ -93,3 +93,9 @@ check-ci = {sequence = ["lint", "unit-test-with-cov"], help = "Lint and run unit
 
 # Build and check
 pre-push = {sequence = ["build", "check-local"], help = "Run all build and check tasks."}
+
+[tool.airbyte_ci]
+optional_poetry_groups = ["dev"]
+poetry_extras = ["file-based", "sphinx-docs", "vector-db-based"]
+poe_tasks = ["build", "check-ci"]
```

Comment:
`check-ci` does not run type checks but they do run in `python_cdk_tests` right? Is it safe to not run type checks in CI for the CDK? ( cc. @girarda )

Reply:
Did we give up on running mypy on modified files? It would feel like a regression to me

Reply:
Yeah, I think we should run type checks still.

Reply:
The type checks currently run and will continue to run as a separate workflow, `run-mypy-on-modified-cdk-files.yml`